cmd/bucket-quota.go

```diff
@@ -155,6 +155,9 @@ func enforceBucketQuota(ctx context.Context, bucket string, size int64) error {
 		return nil
 	}
 
+	if q.Type == madmin.FIFOQuota {
```

Comment:
Types are different here @poornas can you take a look?

Reply:
both are madmin.QuotaType